test/integration/integration_test.cc

```diff
@@ -435,7 +435,7 @@ name: matcher
 
 // This is a regression for https://github.com/envoyproxy/envoy/issues/2715 and validates that a
 // pending request is not sent on a connection that has been half-closed.
-TEST_P(IntegrationTest, UpstreamDisconnectWithTwoRequests) {
+TEST_P(IntegrationTest, DEPRECATED_FEATURE_TEST(UpstreamDisconnectWithTwoRequests)) {
```

Comment:
I don't think this is deprecated. Revert.

Reply:
Change implementation for this, to not warn as a deprecated field.

Reply:
You need to update the test to use the new field, not mark it as deprecated.

/wait

Reply:
Done. Please take a look.